tests/test_views_groups.py

```diff
@@ -34,6 +34,23 @@ def test_groups_can_be_posted_without_id(self):
         self.assertIn('id', resp.json['data'])
         self.assertEqual(resp.json['data']['members'], ['fxa:user'])
 
+    def test_groups_can_be_put_with_empty_body(self):
+        self.app.put('/buckets/beers/groups/simple', headers=self.headers)
+
+    def test_groups_can_be_created_with_just_permissions(self):
+        group = {'permissions': {'write': ['github:me']}}
+        resp = self.app.put_json('/buckets/beers/groups/g',
+                                 group,
+                                 headers=self.headers)
+        self.assertNotIn('members', resp.json['data'])
+
+    def test_groups_can_be_create_without_members_attribute(self):
+        group = {'data': {'alias': 'admins'}}
+        resp = self.app.put_json(self.record_url,
+                                 group,
+                                 headers=self.headers)
+        self.assertEqual(resp.json['data']['members'], [])
+
```

Comment:
Maybe this is stupid, but I think a test to check if we don't overwrite groups on PATCH would be good.